### PR TITLE
Document generic host shutdown timeout

### PR DESCRIPTION
**Pull request by andreasohlund**
And what users needs to to in order to honour it in Version 8

### CODE REVIEW

**Comment by andreasohlund** [comment]
@DavidBoike this is what we said yesterday right?

File: nservicebus/hosting/extensions-hosting_shutdown-timeout_core_[8,).partial.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Shutdown timeout
+
+The Microsoft generic host has a [configurable shutdown timeout that is defaulted to five seconds](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host#shutdowntimeout). NServiceBus will by default wait until all in-flight messages complete and to make sure that the shutdown timeout is honored all message handlers needs to [observe the cancellation token that is available on the message handler context](LINK TO NEW DOCO PAGE).
```
Comment (DavidBoike) [inline nservicebus/hosting/extensions-hosting_shutdown-timeout_core_[8,).partial.md:3]:
```suggestion
The Microsoft generic host has a [configurable shutdown timeout that is defaulted to five seconds](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host#shutdowntimeout). During this shutdown period, NServiceBus will attempt to wait until all in-flight messages complete. To make sure that the shutdown timeout is honored, all message handlers need to [observe the cancellation token that is available on the message handler context](LINK TO NEW DOCO PAGE).
```

**Comment by DavidBoike** [comment]
Needs a link obviously but :+1:

**Comment by andreasohlund** [comment]
>Needs a link obviously but 👍

Have we started on that page yet?

**Comment by andreasohlund** [comment]
Link fixed so merging this into the main PR now

**Pull request merged**